medicines/api/src/azure_search.rs

```diff
@@ -56,7 +56,15 @@ impl juniper::Context for AzureContext {}
 
 fn get_env(key: &str) -> String {
     match std::env::var(key) {
-        Ok(b) => b,
+        Ok(b) => {
```

Comment:
What is `b` here? Is there a more descriptive name that coud be used here that would make this more obvious?

Reply:
+1

Reply:
could maybe change it to `v` but was aiming for short names for tiny scope